src/components/DevicesView.jsx

```diff
@@ -161,15 +161,11 @@ const MoreMenu = ({ device, onRevoke, onConfigure, isMobile }) => {
 }
 
 const DevicesView = props => {
-  const {
-    t,
-    f,
-    lang,
-    breakpoints: { isMobile }
-  } = props
+  const { t, f, lang } = props
```

Comment:
Since you're removing the `withBreakpoints` HOC, you can also remove the `translate` HOC by using: 
```
  const { t, f, lang } = useI18n()
```

Also, the commit should be tagged as `refactor` not `chore`